environment/bashrc/.bashrc_vm

```diff
@@ -11,8 +11,13 @@
 ##  in the LAP virutal environment.
 ##---------------------------------------------------------------------------##
 
-export dracomodules="gcc cmake python openmpi intel-mkl trilinos superlu-dist metis numdiff numactl parmetis gsl random123 eospac csk ndi libquo"
+export dracomodules="git gcc cmake python openmpi intel-mkl trilinos superlu-dist metis numdiff numactl parmetis gsl random123 eospac csk ndi libquo"
 
+# spellcheck disable=SC2206
```

Comment:
`spellcheck -> shellcheck`  

LOL.

Reply:
LOL - somehow my brain keep auto-correcting this!  Ironic...

Reply:
Ok fixed - goodness... let's see if this works.